ci-operator/step-registry/openshift-pipelines/ginkgo-tests/openshift-pipelines-ginkgo-tests-commands.sh

```
@@ -0,0 +1,29 @@
+# Login to the cluster
+if [ -s "${KUBECONFIG:-}" ]; then
+    oc whoami
+else
+    # Login for ROSA & Hypershift platforms
+    (set +x; eval "$(cat "${SHARED_DIR}/api.login")")
+fi
+
+cd /tmp/release-tests-ginkgo
+
+# Run OpenShift Pipelines e2e tests using Ginkgo label filter
+(set +x
```

Shellcheck (0.11.0)

[error] Couldn't parse this explicit subshell. Fix to allow more checks.

(SC1073)